src/bank_forks.rs

```diff
@@ -23,8 +23,18 @@ impl BankForks {
         self.banks[&self.working_bank_id].clone()
     }
 
+    // TODO: use the bank's own ID instead of receiving a parameter
     pub fn insert(&mut self, bank_id: u64, bank: Bank) {
```

Comment:
With this implementation, this should be renamed to something like `replace`

Reply:
or BankForks should be called BankHeads?

Reply:
But there still ought to be two separate functions right?  `insert` was intended for new forks, and no `replace` existed (because I had no user yet).  I don't think you should be augmenting `insert` to "work" while being misused.

Reply:
how about if I change it when I delete BankForks?

Reply:
sure